apps/extension/client/src/initiateEditor.ts

```diff
@@ -32,6 +42,20 @@ export const initiateEditor = (context: vscode.ExtensionContext) => {
     layoutString: string | undefined,
     implementations: ImplementationsMetadata
   ) => {
+    if (isTelemetryEnabled) {
+      fetch("https://stately.ai/registry/api/analyze", {
+        method: "POST",
+        body: JSON.stringify([
+          {
+            event: "Opening Editor From VS Code",
+            properties: {
+              time: Math.floor(Date.now() / 1000),
+              distinct_id: machineId,
```

Comment:
Is it possible to get some sort of user id from the Code API? If one user opens multiple machines daily, it would skew our DAU count.

Reply:
This is inevitable. The same problem happens in a lot of other situations and we can't easily fight it.